dev/scripts/generate-tarball.sh

```diff
@@ -27,25 +28,31 @@ THIS=$(cd "$( dirname "$0" )"; pwd)
 cd ${THIS}
 TARBALL_DIR="${THIS}/tarballs"
 WORK_DIR="workdir"
+CLIENT_DIR="client"
```

Comment:
Now that I think about it, `client` is fine since we have `core` for our actual code.